meinberlin/apps/budgeting/assets/FilterBarListMapSwitch.jsx

```diff
@@ -6,13 +6,13 @@ export const FilterBarListMapSwitch = ({ query }) => {
   return (
     <div className="btn-group__container">
       <div className="btn-group" role="group">
-        <div
+        <button
```

Comment:
good point, didn't know about `aria-pressed` tbh. Actually I realized this component is a little bit of a fake, because this button never changes its state. E.g. the buttons `View as List` and `View as Maps` will remain always the same in this component.

Why is that? Because we use the Map in a Django template, if we switch to toggle the `Maps` button, then it will show this button as a django template.